assets/scss/_variables.scss

```diff
@@ -44,6 +44,9 @@ $spacer: 1rem;
 $card-border-width: 0;
 $card-border-radius: 2rem;
 $dashboard-header-height: 68px;
+$fs-2rem: 2rem;
+$fs-1-75rem: 2rem;
+$fs-2-5rem: 2.5rem;
```

Comment:
both of them are 2rem ?
can't it be just one of them?

Reply:
i don't see where it is used

Reply:
what i meant was to do this

`@import 'bootstrap/scss/utilities';
$utilities: map.merge($utilities, (
  "fs-2rem": (
    property: font-size,
    class: fs-2rem,
    values: (
      2rem: 2rem
    )
  )
));`

this way you can use 'fs-2rem' as a class directly, so no need for <style> tag in component.